tensorflow/core/kernels/conv_grad_ops_3d.cc

```diff
@@ -1572,8 +1572,12 @@ struct LaunchConvBackpropInputOp<Eigen::bfloat16> {
     // Performant bfloat16 operations are supported for Ampere+ GPUs. For
     // pre-Ampere GPUs, we cast inputs to float and outputs back to bfloat16.
     auto* stream = ctx->op_device_context()->stream();
+#if GOOGLE_CUDA    
     const bool cast_to_float = !stream->GetCudaComputeCapability().IsAtLeast(
         se::CudaComputeCapability::AMPERE);
+#else
+    const bool cast_to_float = false;
```

Comment:
Always set to false for us?

Reply:
I am going to create a ticket to investigate which architecture support bfloat16 operations, then this will be updated.